history/characters/wc_easteregg_characters.txt

```diff
@@ -457,4 +457,281 @@ easteregg13={ #Shrek reference
+	culture = azerothian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 5
+	stewardship = 4
+	intrigue = 7
+	learning = 4
+	#Personality
+	trait = brave
+	trait = gregarious
+	trait = stubborn
+	#Education
+	trait = education_intrigue_2
+	#Other
+	trait = adventurer
+	trait = physique_good_1
+	584.3.19 = { 
+		birth = yes
+		trait = creature_human
+	}
+	600.1.1 = {
+		set_relation_friend = character:easteregg15
+	}
+	# switches from lvl 2-3 rogue into paladin around 610, level 3 holy
+	680.6.12 = { death = yes }
+}
+
+# Part of Eden Aurorae guild, therefore a landless playable character
+easteregg15 = {
+	name = "Yimo"
+	dna = yimo_dna
+	culture = gnome
+	religion = rationalism
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 6
+	martial = 4
+	stewardship = 4
+	intrigue = 5
+	learning = 8
+	#Personality
+	trait = compassionate
+	trait = generous
+	trait = diligent
+	#Education
+	trait = education_learning_3
+	#Other
+	trait = adventurer
+	trait = intellect_good_1
+	trait = magic_good_2
+	466.5.10 = { 
+		birth = yes
+		trait = creature_gnome
+	}
+	# level 3 frost mage
+	608.10.14 = { death = { death_reason = death_battle } }	# Killed in Naxxramas
+}
+
+# Tauren mercenary, therefore a landless playable character
+easteregg16 = {
+	name = "Ball"
+	#dna = ball_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 3
+	martial = 6
+	stewardship = 4
+	intrigue = 3
+	learning = 3
+	#Personality
+	trait = brave
+	trait = stubborn
+	trait = content
+	#Education
+	trait = education_martial_2
+	#Other
+	trait = adventurer
+	trait = physique_good_1
+	547.8.19 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	# level 2 warrior
+	662.2.19 = { death = yes }
+}
+
+# Tauren mercenary, therefore a landless playable character
+easteregg17 = {
+	name = "Jubai"
+	#dna = jubai_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 4
+	stewardship = 3
+	intrigue = 3
+	learning = 6
+	#Personality
+	trait = brave
+	trait = forgiving
+	trait = honest
+	#Education
+	trait = education_learning_3
+	#Other
+	trait = adventurer
+	trait = magic_good_1
+	506.3.10 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	583.1.1 = {
+		give_nickname = nick_the_ancient
+	}
+	# level 3 druid
+	641.6.10 = { death = yes }
+}
+
+# Dwarven officer of Ironforge
+easteregg18 = {
+	name = "Serphentos"
+	dna = serphentos_dna
+	culture = bronzebeard
+	religion = khazism
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 2
+	martial = 7
+	stewardship = 5
+	intrigue = 5
+	learning = 3
+	#Personality
+	trait = vengeful
+	trait = honest
+	trait = brave
+	#Education
+	trait = education_martial_3
+	#Other
+	trait = physique_good_2
+	trait = lifestyle_blademaster
+	377.8.24 = { 
+		birth = yes
+		trait = creature_dwarf
+	}
+	511.7.15 = {
+		employer = 17001 # Magni [4200]
+		give_council_position = councillor_marshal
```

Comment:
How lore accurate is this?

Reply:
made Serp unemployed and moved Monóri to Auberdine